CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project 1.4.x per each release will be documented in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v1.4.0.5] - TBA
+
+### Fixed
+- Failure to accept product startup options
```

Comment:
@kaumini can you please create an appropriate GitHub issue for the changes done, in relation with preserving the content of the directories to be persisted/shared (i.e. specific directories rather than `repository/deployment/server`) and add it as part of the change log changes?

Reply:
Fixed